test/e2e/framework/pod/wait.go

```diff
@@ -536,30 +535,6 @@ func WaitForPodsWithLabelRunningReady(c clientset.Interface, ns string, label la
 	return pods, err
 }
 
-// WaitForPodsInactive waits until there are no active pods left in the PodStore.
-// This is to make a fair comparison of deletion time between DeleteRCAndPods
-// and DeleteRCAndWaitForGC, because the RC controller decreases status.replicas
-// when the pod is inactvie.
-func WaitForPodsInactive(ps *testutils.PodStore, interval, timeout time.Duration) error {
```

Comment:
I don't think this method belongs in the test framework.
/hold

Reply:
That is a nice point.
When I started creating this PR, TBH I felt that this is not good for separation/cleanup for test framework.
So it is fine to drop this and I'd like to find another way to reduce circular dependencies.